docs/user-guide/01-blueprint-reference.md

```diff
@@ -1521,6 +1521,67 @@ network --bootproto=dhcp --device=link --activate --onboot=on
 Note that osbuild-composer will automatically add the command that installs the system (`liveimg` or `ostreesetup`) if one is relevant for the image type (`image-installer` and `edge-installer`/`iot-installer` respectively), so this line or any line that conflicts with it should not be included. See the relevant [Kickstart documentation](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer) for more information.
 This customization cannot be used in combination with any other installer customizations.
 
+#### Anaconda Modules 🟣 {#installer-kickstart}
+
+The Anaconda installer can be configured by enabling or disabling its dbus modules.
+
+<Tabs values={tabValues} >
```

Comment:
Please change to `tabValuesBootcOnly` and make `bootc` the default
```suggestion
<Tabs values={tabValues} defaultValue="bootc">
```

where you need to implement this in `src/components/Highlight.js` first and import here 😅
```
export const tabValuesBootcOnly = [
    { label: '⚪ on premises', value: 'on-premises', attributes: { className: 'highlight tab disabled' } },
    { label: '⚪ hosted', value: 'hosted', attributes: { className: 'highlight tab disabled' } },
    { label: '🟣 bootc', value: 'bootc', attributes: { className: 'highlight tab bootc' } },
];
```

Reply:
Very nice, thanks.  Added!